.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
It looks like you have committed some local IDE settings. I would suggest omitting these so others don't pull these down.